File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleOp/ServoTester.java
package org.firstinspires.ftc.teamcode;

import com.qualcomm.robotcore.eventloop.opmode.OpMode;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
import com.qualcomm.robotcore.hardware.Servo;

@TeleOp
public class ServoTester extends OpMode {
    //declare all motors
    Servo rightClaw;
    Servo leftClaw;

    @Override
    public void init() {
        telemetry.addData("Status", "Initializing");

        //assign all motors
        rightClaw = hardwareMap.get(Servo.class, "RightClaw");
        leftClaw =  hardwareMap.get(Servo.class, "LeftClaw");

        //tells user that the motors have been initialized
        telemetry.addData("Status", "Initialized!");
    }

    @Override
    public void loop() {
        if(gamepad1.dpad_right) {
            rightClaw.setPosition(1);
            leftClaw.setPosition(1);
        }
        if(gamepad1.dpad_left) {
            rightClaw.setPosition(0);
            leftClaw.setPosition(0);
        }
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Teleop/ServoTester.kt
package org.firstinspires.ftc.teamcode.Teleop

import com.qualcomm.robotcore.eventloop.opmode.Disabled
import com.qualcomm.robotcore.eventloop.opmode.TeleOp
import com.qualcomm.robotcore.hardware.Servo
import org.firstinspires.ftc.teamcode.DriveMethods

@Disabled
@TeleOp(name = "ServoTester", group = "Testing")
class ServoTester: DriveMethods() {
    override fun runOpMode() {
        initMotorsSecondBot()
        initSlideMotors()
        val servoBeingTested: Servo = hardwareMap.get(Servo::class.java, "autoServo")!!

        waitForStart()

        servoBeingTested.position = 0.1
        while (opModeIsActive()) {
            if (gamepad2.y) {
                servoBeingTested.position+=.04
                sleep(150)
            }

            if (gamepad2.x) {
                servoBeingTested.position-=.04
                sleep(150)
            }
            telemetry.addData("Servo Value: ", servoBeingTested.position)
            telemetry.update()
        }
    }
}